content/en/docs/Policy Reports/_index.md

```diff
@@ -97,56 +136,12 @@ kubectl get clusterpolicyreport
 ```
 
 {{% alert title="Tip" color="info" %}}
-For a graphical view of Policy Reports, check out [Policy Reporter](https://github.com/kyverno/policy-reporter#readme).
+Note that the name of the report is mostly random, don't forget to add `-o wide` to  show additional informations that will help identify the resource associated with the report.
 {{% /alert %}}
 
-## Viewing policy violations
-
-Since the report provides information on all rule and resource execution, returning only select entries requires a filter expression.
-
-Policy reports can be inspected using either `kubectl describe` or `kubectl get`. For example, here is a command, requiring `yq`, to view only failures for the (Namespaced) report called `cpol-disallow-privileged-containers`:
-
-```sh
-kubectl get polr cpol-disallow-privileged-containers -o jsonpath='{.results[?(@.result=="fail")]}' | yq -p json -
-```
-
-```yaml
-category: Pod Security Standards (Baseline)
-message: 'validation error: Privileged mode is disallowed. The fields spec.containers[*].securityContext.privileged and spec.initContainers[*].securityContext.privileged must be unset or set to `false`.          . rule privileged-containers failed at path /spec/containers/0/securityContext/privileged/'
-policy: disallow-privileged-containers
-resources:
-  - apiVersion: v1
-    kind: Pod
-    name: h0nk
-    namespace: default
-    uid: 71a4a8c8-8e02-46ed-af3d-db38f590a5b6
-result: fail
-rule: privileged-containers
-scored: true
-severity: medium
-source: kyverno
-timestamp:
-  nanos: 0
-  seconds: 1.666094801e+09
----
-category: Pod Security Standards (Baseline)
-message: 'validation error: Privileged mode is disallowed. The fields spec.containers[*].securityContext.privileged and spec.initContainers[*].securityContext.privileged must be unset or set to `false`.          . rule privileged-containers failed at path /spec/containers/0/securityContext/privileged/'
-policy: disallow-privileged-containers
-resources:
-  - apiVersion: v1
-    kind: Pod
-    name: badpod
-    namespace: default
-    uid: 8ef79afd-f2b4-44f8-8c50-dabdda45b8c0
-result: fail
-rule: privileged-containers
-scored: true
-severity: medium
-source: kyverno
-timestamp:
-  nanos: 0
-  seconds: 1.666095335e+09
-```
+{{% alert title="Tip" color="info" %}}
+For a graphical view of Policy Reports, check out [Policy Reporter](https://github.com/kyverno/policy-reporter#readme).
```

Comment:
Do you think the docs would be a better link target?
https://kyverno.github.io/policy-reporter/